js/oppfolgingsdialogNpm/oppfolgingsdialogerSagas.js

```diff
@@ -14,7 +19,7 @@ export function* hentOppfolgingsdialoger() {
     yield put(henterOppfolgingsdialoger());
 
     try {
-        const url = `${process.env.REACT_APP_OPPFOELGINGSDIALOGREST_ROOT}/sykmeldt/oppfoelgingsdialoger`;
+        const url = `${hentSyfoApiUrl(API_NAVN.SYFOOPPFOLGINGSPLANSERVICE)}/arbeidstaker/oppfolgingsplaner`;
```

Comment:
Du går fra sykmeldt -> arbeidstaker her. Er det med vilje?

Reply:
nvm 🙈

Reply:
Det var med vilje da jeg lagde nytt endepunkt, for å forenkle. Men burde vel egentlig ikke ha gjort det ettersom sykmeldt ikke må være en arbeidstaker.